docker-compose.ci.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.4'
```

Comment:
This is ok to use Docker Compose. But if you don't want to maintain that, you can also use the Postgres server which is provided directly in the GitHub Actions environment: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#postgresql

Reply:
So you can probably remove this file now.